Sources/Data.swift

```diff
@@ -28,7 +28,7 @@ extension Data: RangeReplaceableCollection, MutableCollection {
     }
 
     #if swift(>=3.0)
-        public mutating func replaceSubrange<C : Collection where C.Iterator.Element == Byte>(subRange: Range<Int>, with newElements: C) {
+        public mutating func replaceSubrange<C : Collection where C.Iterator.Element == Byte>(_ subRange: Range<Int>, with newElements: C) {
```

Comment:
@Obbut upgrading Swift versions is going to force us to do a minor bump anyway, so we might as well get this out of the way.